__tests__/app.test.js

```diff
@@ -65,6 +65,52 @@ describe("GET /api/articles", () => {
         expect(body).toBeSortedBy("created_at", { descending: true });
       });
   });
+
```

Comment:
A couple of tests that could have been added first would be:

To assert that a query with just sort by works as expected:
E.g `/api/articles?sort_by=votes`

To assert that a query with just order works as expected:
E.g `/api/articles?order=asc`